src/SupportView.ts

```diff
@@ -93,6 +93,11 @@ export default class SupportView extends EventHandler {
   }
 
   private addDefaultColumn() {
+
+// Check the support view for matrix
```

Comment:
since there is now a subclass called `MatrixSupportView` I would prefer a cleaner solution. 

e.g. 
mark this `addDefaultColumn` method as protected and override it in the `MatrixSupportView` with and empty body